src/objects/zcl_abapgit_object_suso.clas.abap

```diff
@@ -167,29 +167,47 @@ CLASS zcl_abapgit_object_suso IMPLEMENTATION.
       co_act_delete TYPE activ_auth VALUE '06'.
 
     DATA:
-      lv_act_head TYPE cl_suso_gen=>td_act,
-      lv_dummy    TYPE string,
-      lo_suso     TYPE REF TO cl_suso_gen,
-      lv_failed   TYPE abap_bool.
-
-    CREATE OBJECT lo_suso.
-
-    lv_failed = lo_suso->suso_load_from_db( mv_objectname ).
+      lv_act_head            TYPE activ_auth,
+      lv_dummy               TYPE string,
+      lo_suso                TYPE REF TO object,
+      lv_failed              TYPE abap_bool,
+      lv_suso_collect_in_cts TYPE i.
+
+    " Downport: CL_SUSO_GEN doesn't exist in 702
+    CREATE OBJECT lo_suso
```

Comment:
SUSO objects also exist in 702, so this code can be triggered, which will cause a dump?

Reply:
Sure, as a first step I'd like to get rid of the syntax error.